src/core/services/model.stringify.js

```diff
@@ -0,0 +1,50 @@
+import {flatten} from '../utility';
+
+export function stringifyFactory(property) {
+	switch (property) {
+		case 'filter':
+			return filter;
+		case 'sort':
+			return sort;
+		case 'group':
+		case 'pivot':
+			return transformByKeys(property);
+		default:
+			throw new AppError(
```

Comment:
can we just return function that returns empty? becuase user can persist custom settings not only filter, sort, group and pivot